src/lua/lua_thread_pool.h

```diff
@@ -53,7 +53,7 @@ lua_thread_pool_free (struct lua_thread_pool *pool);
 
 /**
  * Extracts a thread from the list of available ones.
- * It immediately becames running one and should be used to run a Lua script/function straight away.
+ * It immediately becomes running one and should be used to run a Lua script/function straight away.
```

Comment:
fwiw, this should be `the running one`, but that's outside the scope of the changes here.

Reply:
Yes, sure.